content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,313 @@
+
+## Requirements
+
+### Clusters
+
+This guide requires that you have two Kubernetes clusters with any of the
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
+
+The first cluster contains the {{< gloss >}}external control plane{{< /gloss >}} installed
+in the `external-istiod` namespace. This Istio control plane serves as the external control plane for the second cluster.
+An ingress gateway is installed in the `istio-system` namespace. The ingress gateway provides mesh sidecars access to the
+external istiod in the `external-istiod` namespace.
+
+The second cluster is a {{< gloss >}}remote cluster{{< /gloss >}} hosting the mesh. Its Kubernetes API server also provides
+the configuration for the control plane (istiod) running in the external cluster.
+
+### API Server Access
+
+The API Server in the remote config cluster must be accessible to the external control plane cluster. Many cloud providers make API Servers publicly accessible via network
+load balancers (NLBs). If the API Server is not directly accessible, you will
+have to modify the installation procedure to enable access. For example, the
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) gateway used in
+the multi-network and primary-remote configurations could also be used
+to enable access to the API Server.
+
+## Environment Variables
+
+This guide will refer to two clusters named `external_cluster` and `remote_cluster`. The following environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_EXTERNAL_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the external control plane cluster.
+`CTX_REMOTE_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the remote cluster.
+`EXTERNAL_ISTIOD_ADDR` | The host name for the `remote_cluster` to access the external control plane.
+`SSL_SECRET_NAME` | The secret name used to access the ingress gateway on the external control plane cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_EXTERNAL_CLUSTER=external_cluster
+$ export CTX_REMOTE_CLUSTER=remote_cluster
+$ export REMOTE_ISTIOD_ADDR=myexternal-istiod.cloud.com
+$ export SSL_SECRET_NAME=myexternal-istiod-secret
```

Comment:
Can not remember correctly about sds flow, should this secret be mounted to gateway in order to serve sds?

Reply:
This secret is specified later on in the gateway resource configuration, and the gateway will attempt to load the secret dynamically.